cirq/aqt/aqt_device.py

```diff
@@ -221,13 +203,38 @@ def get_aqt_device(num_qubits: int) -> Tuple[IonDevice, List[LineQubit]]:
     return ion_device, qubit_list
 
 
-def get_default_noise_dict() -> Dict[str, Any]:
+def get_aqt_default_noise_dict() -> Dict[str, Any]:
     """Returns the current noise parameters"""
     default_noise_dict = {
-        'X': ops.depolarize(1e-3),
-        'Y': ops.depolarize(1e-3),
-        'Z': ops.depolarize(1e-3),
-        'MS': ops.depolarize(1e-2),
+        'X': depolarize(1e-3),
+        'Y': depolarize(1e-3),
+        'Z': depolarize(1e-3),
+        'MS': depolarize(1e-2),
         'crosstalk': 0.03
     }
     return default_noise_dict
+
+
+def get_aqt_device_op_string(op_obj: Operation) -> str:
```

Comment:
Is this needed for users?  If so, why?  

If not, I would change it to _get_aqt_device_op_string to denote that it is private.